Source/Times.cs

```diff
@@ -168,17 +174,83 @@ internal string GetExceptionMessage(string failMessage, string expression, int c
 		{
 			return string.Format(
 				CultureInfo.CurrentCulture,
-				this.messageFormat,
+				Resources.NoMatchingCalls,
 				failMessage,
-				expression,
-				this.from,
-				this.to,
-				callCount);
+				this.evaluator.GetExceptionMessage(callCount),
+				expression);
 		}
 
 		internal bool Verify(int callCount)
 		{
-			return this.evaluator(callCount);
+			return this.evaluator.Verify(callCount);
+		}
+
+		private sealed class RangeExclusiveBasedTimesEvaluator : TimesEvaluator
```

Comment:
Are you OK with nested private classes? Should this and the other one (starting at line 222 below) be placed in a separate code file?